### DISCUSSION TITLE
Hardware For this application

### DISCUSSION THREAD

**Discussion by jorgelamo** [p10tyr/rtsp-to-onvif#13] [Q&A]
If I'm using the Pi just for this purpose, which version would you recommend? I'll be using it to integrate 14 cameras into my unifi-protect system. They are all 1080 TVI hooked up to an encoder. As they fail, I'll replace them with a unified camera.

**Comment by eoprede** [comment]
It may be slightly more complicated to run this on Pi (arm platform) as opposed to any kind of x86 platform (i.e. old laptop). The container is build only for amd64 and you'd have to install an emulator to run it on Pi, which will slow it down. And it doesn't work 100% of the time.
Alternatively, you can just install node with nmp and run the application without container, but you may have to manually restart the app if your Pi turns off/restarts. I'm sure there's a way, but I can't make it work for some reason.
